manifests/install.pp

```diff
@@ -3,7 +3,11 @@
 # Class to install xtrabackup from packages
 #
 class xtrabackup::install {
-  if $::lsbdistcodename != 'lucid' {
+  if $xtrabackup::compatibility__version == 'mysql80' {
```

Comment:
extra underscore here?

Reply:
yes good catch, removed the extra character. Thank you!